.github/workflows/ci.yml

```diff
@@ -50,40 +50,41 @@ jobs:
                 name: "Validate composer.json"
                 run: "composer validate --strict --no-check-lock"
 
-    php-cs-fixer:
-        name: "PHP-CS-Fixer (${{ matrix.php-version }})"
-        needs:
-            - "composer-validate"
-        runs-on: "ubuntu-latest"
-
-        strategy:
-            matrix:
-                php-version:
-                    - '7.3'
-
-        steps:
-            -
-                name: "Checkout code"
-                uses: "actions/checkout@v2"
-
-            -
-                name: "Install PHP"
-                uses: "shivammathur/setup-php@v2"
-                with:
-                    coverage: "none"
-                    extensions: "${{ env.REQUIRED_PHP_EXTENSIONS }}"
-                    php-version: "${{ matrix.php-version }}"
-                    tools: composer:v2
-
-            -
-                name: "Composer install"
-                uses: "ramsey/composer-install@v1"
-                with:
-                    composer-options: "--no-scripts"
-
-            -
-                name: "Run friendsofphp/php-cs-fixer"
-                run: "vendor/bin/php-cs-fixer fix --dry-run --diff --verbose"
+    # Does not support 8.1 yet (but soon) see https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues/5891
```

Comment:
could also dynamically include it or just have a separate job that only runs on 8.0

this is what i do in Phpactor:

https://github.com/phpactor/phpactor/blob/master/.github/workflows/ci.yml#L153